vault/raft.go

```diff
@@ -815,11 +821,12 @@ func (c *Core) getRaftChallenge(leaderInfo *raft.LeaderJoinInfo) (*raftInformati
 		return nil, fmt.Errorf("error decoding raft bootstrap challenge: %w", err)
 	}
 
-	return &raftInformation{
+	raftInfo := &raftInformation{
```

Comment:
This change seem unnecessary.

Reply:
This is yet another case where debugging of the object being returned was required, so the intermediate variable was necessary. I can put it back.